prov/efa/test/efa_unit_test_ep.c

```diff
@@ -168,18 +163,10 @@ void test_rxr_ep_handshake_exchange_host_id(struct efa_resource **state, uint64_
 	will_return(efa_mock_ibv_read_vendor_err_return_mock, FI_EFA_ERR_OTHER);
```

Comment:
Just an observation, the "Setup CQ" code seems like we could probably export it into a function. I assume we do this in multiple places

Reply:
Yes. We do similar mocking in a few places. My assessment on this particular case is that it's probably not easy to generalize and extract to a common function, unless we make that function accept a long list of function pointers(mocks). The other challenge is to mock ordered function calls.